blobs.go

```diff
@@ -185,6 +190,16 @@ type BlobWriter interface {
 	Reader() (io.ReadCloser, error)
 }
 
+// BlobKeeper combines the operations needed for registry's upkeeping such as
+// blob access, read, enumeration and delete. Can be used to prune unnecessary
+// objects.
+type BlobKeeper interface {
```

Comment:
`storage.blobStore` doesn't implement `BlobIngester` nor `BlobStatter` that are contained by `BlobStore`. Likewise `blobStatter` does just a part of the job. Instead of having multiple "getters" like `GetBlobEnumeratorFor(namespace)`, `GetBlobDeleterFor(namespace)` and `GetBlobProviderFor(namespace)`, I'd prefer having just one interface (like `RegistryBlobKeeper(namespace)`) that allows for image pruning. It's not necessary but handy to use.